packages/storage-plus/Cargo.toml

```diff
@@ -20,6 +21,7 @@ bench = false
 cosmwasm-std = { version = "1.0.0", default-features = false }
 schemars = "0.8.1"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
+cw-storage-macro = { version = "0.13.4", optional = true, path = "../storage-macro" }
```

Comment:
Wouldn't this introduce circular deps? I guess `optional = true` breaks that, right?

Reply:
Afaik no, cw-storage-plus imported in cw-storage-macro package is only for tests as it is included in dev-dependencies.
https://github.com/CosmWasm/cw-plus/blob/3f29e20cf1f39edc924b62e0d0296cfdb61f5f1b/packages/storage-macro/Cargo.toml#L19